wiki/markdown/compatibility.md

```diff
@@ -1,38 +1,53 @@
 # Compatibility
 
-We recommend using the latest version of back-end and front-end whenever possible. However, this page contains a compatibility matrix and notes when pairing different versions.
-
-|                      |Front-end ➡           |                      |                      |                      |
-|---------------------:|:--------------------:|:--------------------:|:--------------------:|:--------------------:|
-|**Back-end ⬇**        |                 1.0.0|                 1.0.1|                 1.0.2|                 1.0.3|
-|[1.0.0](#back-end-100)|[✔️️](#front-end-100)   |[✔️️](#front-end-101)   |[✅](#front-end-102)  |[✅](#front-end-103)  |
-|[1.0.1](#back-end-101)|[✔️️](#front-end-100-1) |[✔️️](#front-end-101-1) |[✅](#front-end-102-1)|[✅](#front-end-103-1)|
-|[1.0.2](#back-end-102)|[❕](#front-end-100-2)|[✅](#front-end-101-2)|[✔️️](#front-end-102-2) |[✔️️](#front-end-103-2) |
+We recommend using the latest version of back-end and front-end whenever 
+possible. However, this page contains a compatibility matrix and notes 
+when pairing different versions.
+
+|                      |Recommended front-end versions  |
```

Comment:
I don't think "double" headers are needed now that we just have a tradional table and not a matrix. No need to have the arrow pointing down either in Back-end header.

Reply:
Fix the table headers